dask/array/fft.py

```diff
@@ -86,19 +89,29 @@ def _ihfft_out_chunks(a, n, axis):
     return chunks
 
 
-fft = _fft_wrap(npfft.fft, np.complex_, _fft_out_chunks)
-
-
-ifft = _fft_wrap(npfft.ifft, np.complex_, _fft_out_chunks)
-
-
-rfft = _fft_wrap(npfft.rfft, np.complex_, _rfft_out_chunks)
-
-
-irfft = _fft_wrap(npfft.irfft, np.float_, _irfft_out_chunks)
-
-
-hfft = _fft_wrap(npfft.hfft, np.float_, _hfft_out_chunks)
+def fft_wrapper(fft_func, kind):
+    """ Wrapper of 1D complex FFT functions
 
+    Takes a function that behaves like ``numpy.fft`` functions.
+    """
 
-ihfft = _fft_wrap(npfft.ihfft, np.complex_, _ihfft_out_chunks)
+    if kind == "fft":
+        return _fft_wrap(np.complex_, _fft_out_chunks)(fft_func)
+    elif kind == "ifft":
+        return _fft_wrap(np.complex_, _fft_out_chunks)(fft_func)
+    elif kind == "rfft":
+        return _fft_wrap(np.complex_, _rfft_out_chunks)(fft_func)
+    elif kind == "irfft":
+        return _fft_wrap(np.float_, _irfft_out_chunks)(fft_func)
+    elif kind == "hfft":
+        return _fft_wrap(np.float_, _hfft_out_chunks)(fft_func)
+    elif kind == "ihfft":
+        return _fft_wrap(np.complex_, _ihfft_out_chunks)(fft_func)
```

Comment:
In the spirit of keeping code as data when possible, I recommend creating a dict like 

```python
{'fft': _fft_out_chunks,
 'ifft': _fft_out_chunk, # also should this be ifft?
 ....
}
```